NEWS.md

```diff
@@ -14,6 +14,8 @@ v1.21.1, Fri July 26, 2024
 
 - Disable ZE IPC due to possible memory corruption
 - Fix an error in the error handling path of fi_param_define()
+- Fix integer overflow in ofi_get_mem_size
+- Fix overflow issue in ofi_rbinit
```

Comment:
The newer items are supposed to be added on top. Same for the shm provider.

Reply:
Updated